person.rb

```diff
@@ -19,6 +20,10 @@ def correct_name
     @name
   end
 
+  def rentals(book, date)
+    Rental.new(date, book, self)
+  end
```

Comment:
- [ ] Hi, Great job for making a method for adding rentals. But the requirements say that: `Create the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental`. Therefore I advise you to
- Go back to your code to create a `class Rental`
- Add a relationship between Person and Book using `class Rental`